Makefile

```
@@ -9,41 +9,35 @@ vap_test_files:=\
-	done; 
-
-download: wasm-download kubescape-download controls-download
-
-build: 
-	GOOS=js GOARCH=wasm go -C go build -ldflags="-s -w" -o ../dist/main.wasm cmd/main.go 
+.PHONY: wasm-download kubescape-cel-admission-library-download kubescape-rego-download download build local
+
+wasm-download:
+	curl -fL https://raw.githubusercontent.com/golang/go/refs/heads/master/lib/wasm/wasm_exec.js -o src/wasm/wasm_exec.js
+
+kubescape-cel-admission-library-download:
```

checkmake (0.3.2)

[warning] Target body for "kubescape-cel-admission-library-download" exceeds allowed length of 5 lines (9).

(maxbodylength)